config.py

```diff
@@ -23,7 +23,8 @@
 MODEL_META_DATA = {
```

Comment:
Should we just move this to the ModelWrapper? I don't see this being used anywhere else.

Reply:
Sure